qiskit/schemas/qobj_schema.json

```diff
@@ -153,6 +154,26 @@
                     ],
                     "title": "Frame change pulse"
                 },
+                {
+                    "properties": {
+                        "name": {
+                            "enum": [
+                                "setf"
+                            ]
+                        },
+                        "frequency": {
+                            "OneOf":[
+                            {"type": "number"},
+                            {"type": "string"}]
```

Comment:
Should frequency be allowed to be a string?

Reply:
Not sure, I based this off of the Frame Change pulse